src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -215,8 +217,9 @@ private boolean loadConfig() {
         }
     }
 
-    private boolean extractFile(ResourceLocation resource, File destination) {
-        try(InputStream input = mc.getResourceManager().getResource(resource).getInputStream()) {
+    private boolean extractFile(String resourcePath, File destination) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream input = classloader.getResourceAsStream(resourcePath)) {
```

Comment:
I'll add a second comment after looking around: You never appeared to have altered the use point, which means that, even if it 'works' in your test setup, it does not appear to be looking for the default files in the correct place for the full packaged version -- they're under `jar/assets/inventorytweaks/` not just `jar/`